packages/digitalproperty-app/lib/cmds/landregistry_cmds/listSales.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const LandRegistry = require('./../../landRegistry.js');
+const winston = require('winston');
+const LOG = winston.loggers.get('application');
+
+exports.command = 'records';
+exports.desc = 'Lists all the sales held in the Sales Regsitry';
```

Comment:
list all the sales *records* hled in the Sales *Record* registry

(are they sales records or agreements?  not checked the model but double check it's consistent)